app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  after_create :update_posts_counter
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    User.find(author.id).increment!(:posts_counter)
+  end
+
+  def recent_comments
+    Comment.where(post: post.id).limit(5)
+  end
```

Comment:
- [ ] KIndly make sure to return the most recent comments by first ordering them by date in descending order. you can refer to below code snippets for help. you got this :+1: 

```# app/models/user.rb


def recent_post
  posts.order('created_at Desc').limit(3)
end

```

Reply:
Though you did a mistake here its supposed to be
 def recent_comments
    comments.order('created_at Desc').limit(5)
  end